src/main/resources/js/script.js

```diff
@@ -0,0 +1,61 @@
+var map;
+var latlngs;
+var polygon;
+var markers;
+var markerCount;
```

Comment:
If you use prototype or classical inheritance then you don't need to define global variables. Just look into it if you can refactor it